### ISSUE TITLE
compatibility with numpy 1.10

### ISSUE THREAD

**Issue by m-kreuzer** [brian-rose/climlab#2]
integrating an EBM with numpy 1.10 gives following error message:

> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "climlab/process/time_dependent_process.py", line 149, in integrate_years
>     self.timeave[varname] /= numsteps
> TypeError: ufunc 'divide' output (typecode 'l') could not be coerced to provided output parameter (typecode '?') according to the casting rule ''same_kind''

This happens when the function *integrate_years()* tries to divide the boolean array *self.timeave['ice']* by *numstep* .

The error occurs with numpy version 1.10. Previous versions of numpy are running fine.

**Comment by brian-rose** [comment]
Thanks Moritz. 
I encountered the same error. The issue is that one particular diagnostic called `ice` is a boolean rather than a number. The error arises when we try to calculate time-averages of all diagnostic quantities.  `numpy` has apparently changed its casting rules and is no longer willing to divide a boolean by an integer.

I got rid of this issue in the newer `climlab` code (branch `newapi`) by getting rid of any non-numerical diagnostics. So the problem will go away when `climlab 0.3` is released (most likely in January). The work-around is to use `numpy 1.9`, or modifying the EBM code to avoid saving the boolean diagnostic.
